ow_plexil/src/plexil-adapter/joint_support.h

```diff
@@ -7,41 +7,61 @@
 
 // Support for lander joints, based on ROS /joint_states message.
 
+// NOTE: This package (ow_plexil) is implemented for the lander in the
+// ow_simulator package.  Specifically, the joints and joint ordering
+// (in /joint_states) is assumed fixed, and is often hardcoded.
+
 #include <string>
 
-enum class Joint {
-  shoulder_yaw,
-  shoulder_pitch,
-  proximal_pitch,
-  distal_pitch,
-  hand_yaw,
-  scoop_yaw,
-  antenna_pan,
-  antenna_tilt,
-  grinder
+const size_t KnownJoints = 9;
+
+enum Joint {
+  // NOTE: we could use the names in /joint_states, but these seem more readable.
+  ANTENNA_PAN = 0,
+  ANTENNA_TILT = 1,
+  DISTAL_PITCH = 2,
+  GRINDER = 3,
+  HAND_YAW = 4,
+  PROXIMAL_PITCH = 5,
+  SCOOP_YAW = 6,
+  SHOULDER_PITCH = 7,
+  SHOULDER_YAW = 8
```

Comment:
This ordering appears not to change, but I'm also unsure what it is set by to begin with. That makes me nervous. A change in `ow_lander` could unexpectedly cause this ordering to break.

Reply:
Good question.  @mogumbo, do you know what gives the joints their particular order in /joint_states, and what could make the order change?

Reply:
The names in /joint_states are alphabetized, which is probably a good move by ROS so you're never in doubt about the order. I'm not aware of a way to change the order. These names used in ow_plexil are not exactly the same as the ones in /joint_states, but they do happen to be in the same order as /joint_states when alphabetized.

Reply:
That is very good to know. So this ordering should be fine so long as we don't modify the joints list. @kmdalal Can we have the comment at the top modified so that it mentions the alphabetical nature of joint_states?

Reply:
Yes, good info.  I added a descriptive comment.